addon/styles/_frost-button.scss

```diff
@@ -83,6 +83,18 @@ $frost-color-button-info-content-subtext-color: rgba($frost-color-white, .8);
       &:not(.disabled) {
         border: solid 2px $frost-color-blue-4;
       }
+
+      &.small {
+        padding: 0 8px;
```

Comment:
Asking a little extra here, but could you make the original padding into CSS variables `$frost-button-small-padding` and then just subtract 2 here?  In case the padding changes at a later stage